src/pipecat/services/tavus.py

```diff
@@ -37,13 +37,15 @@ def __init__(
         replica_id: str,
         persona_id: str = "pipecat0",  # Use `pipecat0` so that your TTS voice is used in place of the Tavus persona
         session: aiohttp.ClientSession,
+        output_sample_rate: int = 16000,
```

Comment:
I'm wondering if we should just call this `sample_rate` as the rest of the services.